README.md

```diff
@@ -448,6 +449,37 @@ Approve a declarative migration plan.
 * `link` - The URL of the plan in [Atlas Registry](https://atlasgo.io/registry).
 * `status` - The status of the plan. For example, `PENDING` or `APPROVED`.
 
+### `ariga/atlas-action/monitor/schema`
+
+Sync the database schema to Atlas Cloud.
```

Comment:
```suggestion
Sync the database schema to Atlas Cloud.
```
From reading this line I can't see the difference between this action and the `schema/push` :)
Maybe mention the name of the object in Atlas Cloud that's being synced somehow:

"Syncs a monitored scope in Atlas Cloud with an actual database schema"